examples/model_selection/gp_search.py

```diff
@@ -0,0 +1,246 @@
+from sklearn.datasets import load_digits
+from sklearn.gp_search import GPSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import fetch_20newsgroups
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+from sklearn.linear_model import SGDClassifier
+from sklearn.grid_search import GridSearchCV, RandomizedSearchCV
+from sklearn.pipeline import Pipeline
+
+import logging
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def extend_result(n_tests, tmp_res):
+    res = np.zeros(n_tests)
+    l = len(tmp_res) - 1
+    for i in range(n_tests):
+        res[i] = tmp_res[min(i, l)]
+
+    return res
+
+
+def test1():
```

Comment:
is this a unittest? if yes, please move it to a separate file in a tests subdirectory. also use some assert_... statements to check for failure conditions etc.